data/cache/__init__.py

```diff
@@ -38,18 +38,16 @@ def get_model_cache(config):
         return cache
 
     if engine == "redis":
-        host = cache_config.get("host", None)
-        if host is None:
-            raise Exception("Missing `host` for Redis model cache configuration")
+        primary_config = cache_config.get("primary_config", None)
+        replica_config = cache_config.get("replica_config", primary_config)
```

Comment:
nit: remove the `_config` suffix from the config key.
The final config should look like:
```
{
  "engine":  "redis",
  "primary": {
    ...
  }
  "replica": {
    ...
  }
}
```

Reply:
👍